editor/components/warning/style.scss

```diff
@@ -10,13 +10,22 @@
 	align-items: center;
 	width: 96%;
 	max-width: 780px;
-	padding: 20px 20px 10px 20px;
+	padding: 20px 20px 0px 20px;
 	background-color: $white;
 	border: 1px solid $light-gray-500;
 	text-align: center;
 	line-height: $default-line-height;
 	box-shadow: $shadow-popover;
 
+	p {
+		font-family: $default-font;
```

Comment:
The addition of these styles makes the identical styles below in `.editor-visual-editor & p {` redundant.